src/main/java/lotto/view/ResultView.java

```diff
@@ -0,0 +1,45 @@
+package lotto.view;
+
+import lotto.domain.LottoGame;
+import lotto.domain.LottoGames;
+import lotto.domain.LottoStatics;
+import lotto.domain.Profit;
+
+import java.io.PrintStream;
+
+public class ResultView {
+    PrintStream view = new PrintStream(System.out);
+    private String ENTER = "\n";
```

Comment:
java 에서 개행 문자는 OS 마다 다를 수 있습니다
다음과 같이 해주면 어떨까요? 😄 
```suggestion
    private String ENTER = System.lineSeparator();
```